packages/react/src/components/Breadcrumb/Breadcrumb.mdx

```diff
@@ -59,5 +72,6 @@ You can use the `isCurrentPage` prop on a `BreadcrumbItem` to represent the curr
 
 ## Feedback
 
-Help us improve this component by providing feedback through, asking questions on Slack,or updating this file on GitHub
-[GitHub](https://github.com/carbon-design-system/carbon/edit/master/packages/react/src/components/Breadcrumb/Breadcrumb.mdx).
\ No newline at end of file
+Help us improve this component by providing feedback through, asking questions
+on Slack,or updating this file on GitHub
```

Comment:
```suggestion
on Slack, or updating this file on GitHub
```

Reply:
This happens in every Feedback section (37x)

Reply:
Ooof, let me fix that